katsdptelstate/memory.py

```diff
@@ -19,16 +19,15 @@
 import bisect
 import re
 import logging
-import os
-
-try:
-    from rdbtools import RdbParser, RdbCallback
-except ImportError as _rdbtools_import_error:
-    RdbCallback = object     # So that _Callback can still be defined
-    RdbParser = None
 
 from .telescope_state import Backend, ImmutableKeyError
 from .rdb_utility import dump_string, dump_zset
+try:
+    from . import rdb_reader
+    from .rdb_reader import RdbCallback
```

Comment:
I'm a bit nervous about importing this from `.rdb_reader` rather than from `rdbtools`. You're effectively making it part of the `.rdb_reader` public API.

Reply:
Agreed. I used to have `from .rdb_reader import RdbCallback, load_from_file` so it was a one-liner, but now it's two lines anyway (since I decided an unqualified `load_from_file` in the namespace is not so nice).